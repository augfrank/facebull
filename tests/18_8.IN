M0   C0   C1  2622
M1   C0   C2  2715
M2   C0   C3  2730
M3   C0   C4  2742
M4   C0   C5  2387
M5   C0   C6  2543
M6   C0   C7  2497
M7   C0   C8  2869
M8   C0   C9  2997
M9   C0   C10  2075
M10   C0   C11  2091
M11   C0   C12  2842
M12   C0   C13  2274
M13   C0   C14  2428
M14   C0   C15  2588
M15   C0   C16  2412
M16   C0   C17  2679
M17   C1   C0  2777
M18   C1   C2  2837
M19   C1   C3  2056
M20   C1   C4  2955
M21   C1   C5  2946
M22   C1   C6  2050
M23   C1   C7  2547
M24   C1   C8  2952
M25   C1   C9  2445
M26   C1   C13  2096
M27   C1   C14  2693
M28   C1   C15  2551
M29   C2   C0  2322
M30   C2   C1  2134
M31   C2   C3  2745
M32   C2   C4  2619
M33   C2   C5  2803
M34   C2   C6  2670
M35   C2   C7  2294
M36   C2   C8  2287
M37   C2   C10  2757
M38   C2   C11  2424
M39   C2   C12  2276
M40   C2   C13  2989
M41   C2   C14  2298
M42   C2   C15  2000
M43   C2   C16  2758
M44   C2   C17  2376
M45   C3   C0  2522
M46   C3   C1  2411
M47   C3   C2  2977
M48   C3   C5  2566
M49   C3   C6  2561
M50   C3   C7  2143
M51   C3   C8  2530
M52   C3   C9  2387
M53   C3   C10  2750
M54   C3   C11  2914
M55   C3   C12  2330
M56   C3   C13  2709
M57   C3   C14  2756
M58   C3   C15  2723
M59   C3   C17  2447
M60   C4   C0  2858
M61   C4   C1  2930
M62   C4   C2  2128
M63   C4   C3  2093
M64   C4   C5  2547
M65   C4   C6  2225
M66   C4   C8  2682
M67   C4   C9  2209
M68   C4   C10  2378
M69   C4   C11  2430
M70   C4   C12  2504
M71   C4   C13  2768
M72   C4   C14  2929
M73   C4   C15  2874
M74   C4   C16  2146
M75   C5   C0  2750
M76   C5   C1  2414
M77   C5   C2  2160
M78   C5   C3  2132
M79   C5   C4  2316
M80   C5   C6  2197
M81   C5   C7  2257
M82   C5   C8  2175
M83   C5   C9  2991
M84   C5   C10  2812
M85   C5   C11  2079
M86   C5   C12  2363
M87   C5   C13  2580
M88   C5   C14  2841
M89   C5   C15  2404
M90   C5   C16  2282
M91   C5   C17  2663
M92   C6   C0  2508
M93   C6   C1  2074
M94   C6   C2  2253
M95   C6   C3  2888
M96   C6   C4  2946
M97   C6   C5  2100
M98   C6   C7  2061
M99   C6   C8  2018
M100   C6   C9  2534
M101   C6   C10  2695
M102   C6   C11  2059
M103   C6   C12  2034
M104   C6   C14  2636
M105   C6   C15  2457
M106   C6   C16  2086
M107   C6   C17  2661
M108   C7   C0  2728
M109   C7   C1  2500
M110   C7   C2  2118
M111   C7   C3  2718
M112   C7   C5  2954
M113   C7   C6  2417
M114   C7   C8  2493
M115   C7   C9  2662
M116   C7   C10  2853
M117   C7   C11  2283
M118   C7   C13  2021
M119   C7   C14  2927
M120   C7   C16  2154
M121   C7   C17  2031
M122   C8   C0  2563
M123   C8   C1  2552
M124   C8   C2  2943
M125   C8   C3  2293
M126   C8   C5  2662
M127   C8   C6  2205
M128   C8   C7  2386
M129   C8   C9  2300
M130   C8   C10  2678
M131   C8   C11  2118
M132   C8   C12  2671
M133   C8   C13  2075
M134   C8   C14  2645
M135   C8   C15  2840
M136   C8   C17  2852
M137   C9   C0  2924
M138   C9   C1  2136
M139   C9   C2  2442
M140   C9   C3  2458
M141   C9   C4  2339
M142   C9   C5  2218
M143   C9   C6  2139
M144   C9   C7  2737
M145   C9   C8  2062
M146   C9   C10  2117
M147   C9   C11  2727
M148   C9   C12  2730
M149   C9   C13  2727
M150   C9   C14  2657
M151   C9   C15  2324
M152   C9   C16  2170
M153   C9   C17  2150
M154   C10   C0  2517
M155   C10   C1  2670
M156   C10   C2  2393
M157   C10   C3  2261
M158   C10   C4  2684
M159   C10   C5  2224
M160   C10   C6  2580
M161   C10   C7  2376
M162   C10   C8  2707
M163   C10   C9  2164
M164   C10   C11  2323
M165   C10   C12  2921
M166   C10   C13  2268
M167   C10   C14  2494
M168   C10   C15  2512
M169   C10   C16  2263
M170   C10   C17  2595
M171   C11   C0  2951
M172   C11   C1  2014
M173   C11   C3  2514
M174   C11   C4  2767
M175   C11   C5  2755
M176   C11   C6  2395
M177   C11   C7  2440
M178   C11   C8  2958
M179   C11   C9  2122
M180   C11   C10  2634
M181   C11   C12  2249
M182   C11   C14  2896
M183   C11   C15  2635
M184   C11   C16  2914
M185   C11   C17  2893
M186   C12   C0  2085
M187   C12   C1  2320
M188   C12   C2  2908
M189   C12   C4  2993
M190   C12   C5  2674
M191   C12   C6  2684
M192   C12   C7  2912
M193   C12   C8  2380
M194   C12   C9  2692
M195   C12   C10  2477
M196   C12   C15  2447
M197   C12   C16  2950
M198   C12   C17  2544
M199   C13   C0  2357
M200   C13   C1  2600
M201   C13   C2  2215
M202   C13   C3  2695
M203   C13   C4  2088
M204   C13   C5  2290
M205   C13   C6  2885
M206   C13   C7  2364
M207   C13   C8  2015
M208   C13   C9  2566
M209   C13   C11  2906
M210   C13   C12  2176
M211   C13   C15  2852
M212   C13   C17  2954
M213   C14   C1  2196
M214   C14   C3  2988
M215   C14   C4  2558
M216   C14   C6  2485
M217   C14   C7  2857
M218   C14   C8  2826
M219   C14   C9  2144
M220   C14   C11  2408
M221   C14   C12  2270
M222   C14   C13  2797
M223   C14   C15  2234
M224   C14   C16  2377
M225   C14   C17  2589
M226   C15   C0  2026
M227   C15   C1  2021
M228   C15   C2  2024
M229   C15   C3  2525
M230   C15   C4  2882
M231   C15   C5  2707
M232   C15   C6  2454
M233   C15   C7  2125
M234   C15   C8  2835
M235   C15   C9  2249
M236   C15   C10  2922
M237   C15   C11  2430
M238   C15   C12  2408
M239   C15   C13  2103
M240   C15   C14  2743
M241   C15   C16  2860
M242   C15   C17  2876
M243   C16   C0  2244
M244   C16   C1  2756
M245   C16   C3  2915
M246   C16   C4  2712
M247   C16   C5  2363
M248   C16   C6  2433
M249   C16   C7  2875
M250   C16   C8  2672
M251   C16   C10  2802
M252   C16   C11  2018
M253   C16   C12  2337
M254   C16   C13  2063
M255   C16   C14  2019
M256   C16   C15  2031
M257   C16   C17  2514
M258   C17   C0  2556
M259   C17   C1  2662
M260   C17   C2  2277
M261   C17   C3  2786
M262   C17   C5  2349
M263   C17   C6  2829
M264   C17   C7  2363
M265   C17   C8  2112
M266   C17   C9  2217
M267   C17   C10  2464
M268   C17   C11  2073
M269   C17   C12  2711
M270   C17   C13  2825
M271   C17   C14  2794
M272   C17   C15  2070
M273   C17   C16  2439
