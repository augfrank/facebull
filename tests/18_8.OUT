37638
9 22 39 50 63 85 106 121 133 145 159 172 186 203 219 228 255 272
